.travis.yml

```diff
@@ -16,3 +16,4 @@ script:
   - go generate -x ./... && git diff --exit-code; code=$?; git checkout -- .; (exit $code) # Check that go generate ./... produces a zero diff; clean up any changes afterwards.
   - go tool vet .
   - go test -v -race ./...
+  - go test -v -tags=integration -run=^$ ./test/integration # Check that integration test builds successfully, but don't run any of the tests (they hit live GitHub API).
```

Comment:
All right.

> Also, I didn't know why `go install` was used instead of `go build`.

For posterity, `go install` and `go build` effectively do the same thing, but the latter tends to throw away its results (unless a single command is provided). In this context, it doesn't matter, but in general, I always prefer to use `go install` since it saves the results of building packages to disk.